templates/system-model.cfg.xml.erb

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- To configure Repose see: http://wiki.openrepose.org/display/REPOSE/Configuration -->
 <system-model xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/system-model/v2.0">
-<%- if ! @rewrite_host_header.nil? %>
+<%- if ! @repose9 %>
+  <%- if ! @rewrite_host_header.nil? %>
```

Comment:
This option still exists in repose 9, it just moved to be on the system-model element itself. As written it's lost entirely here.

Reply:
Tweaked system model template to accommodate this. There wasn't an example of this existing still in the config or in the documented options.